example/distributed_training/README.md

```diff
@@ -0,0 +1,231 @@
+# Distributed Training using Gluon
```

Comment:
Please move this to `docs/tutorials` and change this `README.md` to refer this file instead

Reply:
If I move this to `docs/tutorials`, I can't make it pass the tutorials tests without putting most of the code under "\`\`\`" instead of "\`\`\`python". This will remove syntax highlighting and result is bad user experience. If there is a way to whitelist this tutorial from the automated tests, please let me know.

Reply:
I only see this block not being able to run in a notebook?
```
+```python
+for batch in train_data:
+    # Train the batch using multiple GPUs
+    train_batch(batch, ctx, net, trainer)
+```
```

Reply:
Few more blocks:

```store = mxnet.kv.create('dist’)```
won’t work without a bunch of environment variables being defined. launch.py defines these variables.


```trainer = gluon.Trainer(net.collect_params(),```
won’t work because net is not defined


```print("Total number of workers: %d" % store.num_workers)```
won’t work because store cannot be created. See above.

Reply:
I see, I just don't like the idea of pointing to a github readme for a tutorial from a UI and UX perspective and also have no test coverage on it. I'll let others weight on it, but if that's the only solution... I'm just not happy with it, considering what happened with the tutorial on the straight dope etc. I wonder if we could work out a specific test case where we run a distributed training in a bunch of docker containers.

Reply:
If the concern is about test coverage, can we add distributed_training/cifar10_dist.py to nightly test suite, with some assertion of accuracy on validation set?